ui-tests/specs/home.spec.ts

```diff
@@ -29,3 +32,75 @@ describe("Home Page Tests", () => {
     expect(newPageUrl).toMatch(/contracts$/);
   });
 });
+
+describe("Home Page Overview table Tests", () => {
+  beforeEach(async () => {
+    await browser.url(" ");
+    await waitForPageLoad();
+  });
+
+  it("Should display the overview table with correct data", async () => {
+    const tableData = await HomePage.getTableData(await HomePage.overViewTable);
+    const expectedTableDataFromJson = getLatestInvoicePerVendor().map(
+      (invoice): OverviewTable => {
+        const monthString = prettyMonthName(invoice.month);
+        const reconciliationDetails =
+          generateExpectedBannerDetailsFromPercentagePriceDifference(
+            parseFloat(invoice.price_difference_percentage)
+          );
+        return {
+          contractName: invoice.contract_name,
+          vendor: invoice.vendor_name,
+          month: `${monthString} ${invoice.year}`,
+          reconciliationDetails:
+            reconciliationDetails.bannerMessage.toUpperCase(),
+          details: "View Invoice",
+        };
+      }
+    );
+    const sortTableDataByVendor = tableData
```

Comment:
(minor) Should be sortedTableDataByVendor I think.

Reply:
Thanks Mark, fixed it